server/conf/helper/feature-flags.conf

```diff
@@ -60,3 +60,7 @@ universal_questions = ${?UNIVERSAL_QUESTIONS}
 # Program card images
 program_card_images = false
 program_card_images = ${?PROGRAM_CARD_IMAGES}
+
+# Upsell programs
+suggest_programs_on_application_confirmation_page = false
```

Comment:
Not a comment, more like a question - kinda curious about why we set the bool value twice. does this mean it's default false unless `SUGGEST_PROGRAMS_ON_APPLICATION_CONFIRMATION_PAGE` is not `false`?

Reply:
It's defaulting it to `false` and then overriding with a value from the environment if the environment variable is set. More info here: https://www.playframework.com/documentation/2.9.x/ConfigFile#Substitutions